src/main/scala/com/sumologic/sumobot/http_frontend/RoutingHelper.scala

```diff
@@ -0,0 +1,60 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.sumologic.sumobot.http_frontend
+
+import akka.http.scaladsl.model.headers._
+import akka.http.scaladsl.model.{HttpEntity, HttpRequest, HttpResponse, ResponseEntity}
+import akka.http.scaladsl.model.HttpMethods.{GET, HEAD}
+import akka.stream.Materializer
+
+case class RoutingHelper(origin: String)(implicit materializer: Materializer) {
+  def withAllowOriginHeader(routing: PartialFunction[HttpRequest, HttpResponse]): PartialFunction[HttpRequest, HttpResponse] = {
+    routing.andThen {
+      response: HttpResponse =>
+        val oldHeaders = response.headers.filter(header => header.isNot(`Access-Control-Allow-Origin`.lowercaseName)).toList
```

Comment:
`header => header.` can be simplified to `_.`

Reply:
Also, I think it's more readable if `Not` is appended to `filter` as in `filterNot(...)`

Reply:
> `header => header.` can be simplified to `_.`

`Codacy/PR Quality Review` present in this repository CI marks such usage as error (encountered error from it before, in similar code).